src/test/java/com/amazon/sqs/javamessaging/ExtendedClientConfigurationDeprecatedTest.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2010-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.sqs.javamessaging;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.PutObjectRequest;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import static org.mockito.Matchers.isA;
+import static org.mockito.Mockito.*;
+
+/**
+ * Tests the ExtendedClientConfiguration deprecated functions.
+ */
+public class ExtendedClientConfigurationDeprecatedTest {
+    private static String s3BucketName = "test-bucket-name";
+
```

Comment:
What about having only two test classes and add all the tests there, instead of separate 4 classes?

Reply:
I agree, originally I had just copied the old test files. I have moved all tests using the deprecated functions in the 2 classes and removed the duplicate tests.